service/controller/key/machine_deployment.go

```diff
@@ -20,6 +20,27 @@ func MachineDeploymentDockerVolumeSizeGB(cr infrastructurev1alpha3.AWSMachineDep
 	return strconv.Itoa(cr.Spec.NodePool.Machine.DockerVolumeSizeGB)
 }
 
+func MachineDeploymentContainerdVolumeSizeGB(cr infrastructurev1alpha3.AWSMachineDeployment) string {
+	result, ok := cr.ObjectMeta.Annotations[annotation.AWSContainerdVolumeSize]
+	if !ok {
```

Comment:
why don't we try to parse the value and default if it's invalid like we do for logging volume? This way we could try to create a volume that is "blah" gigs big just by setting a wrong annotation

Reply:
We could try parsing anyway I guess and still return a string number... does that sound better?

Reply:
just did the change, can you check now?